src/many-abci/src/abci_app.rs

```diff
@@ -219,6 +292,17 @@ impl Application for AbciApp {
                     }
                 }
 
+                self.transmitter
+                    .send(transaction_cache::Message::Put(
```

Comment:
And finally, when we _do_ deliver the transaction, we store it in `transaction_cache` so it will be there when we look next time.

Reply:
Yes, correct.